pipeline-CI.yml

```diff
@@ -0,0 +1,23 @@
+trigger:
+- master
```

Comment:
We might want to add batch: true to the trigger to avoid doing this individually for commits that occur in rapid succession